### ISSUE TITLE
method name of get_le() not consistent with torch.optim optimzers

### ISSUE THREAD

**Issue by tanasr** [katsura-jp/pytorch-cosine-annealing-with-warmup#17]
In order to track the learning rate, for example with wandb, one retrieves the last learning rate using, for instance the `CosineAnnealingLR()` with `scheduler.get_last_lr()`. The implementation here is not consistent with the torch naming.
